### PR TITLE
Added implementation for Bitplane Representation of Quantum Images (BRQI)

### PR DESCRIPTION
**Pull request by Dljdd**
## Added implementation for Bitplane Representation of Quantum Images (BRQI) #131 

The implementation of BRQI is present within the embeddings/image_embeddings/brqi.py file.

Added test cases for the BRQI Module in the tests/embeddings/image_embeddings/test_brqi.py file

### CODE REVIEW

**Comment by SaashaJoshi** [comment]
Hey, @Dljdd 
Could you make sure the unit tests and lint are passing?

**Comment by Dljdd** [comment]
Hi, @SaashaJoshi 
I have modified the files, the unit tests and lint should be passing now

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ repos:
   rev: v2.17.0
   hooks:
     - id: pylint
-      args: [--disable=import-error]
+      args: [--disable=import-error, --disable=R0801]
```
Comment (SaashaJoshi) [inline .pre-commit-config.yaml:37]: I cannot accept any changes in this file in this PR. Could you please revert this?

File: demos/brqi.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+
+# BRQI Demo
```
Comment (SaashaJoshi) [inline demos/brqi.md:2]: Could you make a separate PR for the demos?

File: demos/brqi_demo.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import numpy as np
```
Comment (SaashaJoshi) [inline demos/brqi_demo.py:1]: This file must be in a separate PR as well.

**Review by SaashaJoshi — COMMENTED** [review]
Hey, thanks for making the changes, However, I cannot accept any edits in the `.pre-commit-config.yaml` file in this PR. If any changes are required in this file, you will have to make a separate PR for it.

**Comment by SaashaJoshi** [comment]
Hi @Dljdd, I am going to merge this repository. If there are any required changes, I will suggest making another repository. I do not want your hard work go waste by me being late in reviewing your code!
